File: write-sdk.py
import shutil
src = r'C:\Users\account\AppData\Local\Temp\supabase-sdk.js'
dst = r'C:\Users\account\Videos\SchoolSafe V2\app\vendor\supabase-sdk.js'
try:
    shutil.copy(src, dst)
    print('OK')
except Exception as e:
    print('ERR', e)

File: write-sdk.js
const fs = require('fs');
const src = 'C:/Users/account/AppData/Local/Temp/supabase-sdk.js';
const dst = 'C:/Users/account/Videos/SchoolSafe V2/app/vendor/supabase-sdk.js';
try {
    const data = fs.readFileSync(src);
    fs.writeFileSync(dst, data);
    console.log('OK', fs.statSync(dst).size);
} catch (e) {
    console.error('ERR', e.message);
}
